catalog/cataloger.go

```diff
@@ -49,6 +49,9 @@ type Cataloger interface {
 	// CreateRepository create a new repository pointing to 'storageNamespace' (ex: s3://bucket1/repo) with default branch name 'branch'
 	CreateRepository(ctx context.Context, repository string, storageNamespace string, branch string) (*Repository, error)
 
+	// CreateBareRepository create a new repository pointing to 'storageNamespace' (ex: s3://bucket1/repo) with no initial branch or commit
+	CreateBareRepository(ctx context.Context, repository string, storageNamespace string, branch string) (*Repository, error)
```

Comment:
Not sure what parameter `branch` does, the godoc makes me think it shouldn't even be there.

Reply:
gave it a better name.